docs/DeveloperGuide.adoc

```diff
@@ -800,12 +801,22 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
+|`* * *` |user |schedule meetings with my clients |remember when I am supposed to meet them
+
+|`* * *` |user |tag clients with important info like the products they are interested in | so that I can list and group together related clients quickly.
+
+|`* * *` |user |log notes specific to each client |so that I can recall the information when I need it
+
```

Comment:
In the below, you can replace "person" with "client" if you wish, since you have a nice focused target profile and also you have an idea on what problem you are solving for your targetted user.